src/Fork.test.ts

```diff
@@ -24,7 +26,7 @@ describe('Fork', () => {
         return t
       }))
 
-      const t = runSync(f)
+      const t = f.pipe(emptyHandlerContext, runSync)
```

Comment:
It's probably better just to use runAsync.  I'll do that.

Reply:
On second thought, using runSync introduces less machinery into the tests, so seems better for test isolation.